modules/nf-core/samtools/faidx/meta.yml

```diff
@@ -77,4 +93,5 @@ authors:
 maintainers:
   - "@drpatelh"
   - "@ewels"
```

Comment:
```suggestion
```
Harshil has told me before that he might as well be removed from these anyway, and I suspect the same applies to Phil.